dockstore-client/src/test/java/io/dockstore/client/cli/WesToilIT.java

```diff
@@ -0,0 +1,155 @@
+
+    public static final String RUN_ID_PATTERN_PREFIX = "Launched WES run with id:";
+    public static final String COMPLETED_STATE = "state: COMPLETE";
+    public static final String EXECUTOR_ERROR_STATE = "state: EXECUTOR_ERROR";
+    public static final String CANCELED_STATE = "state: CANCELED";
+
+    @Rule
+    public final ExpectedSystemExit systemExit = ExpectedSystemExit.none();
+    @Rule
+    public final SystemErrRule systemErrRule = new SystemErrRule().enableLog();
+    @Rule
+    public final SystemOutRule systemOutRule = new SystemOutRule().enableLog();
+
+    /**
+     * Searches for a runId in the provided string using a pattern printed by the CLI during a launch
+     * @param runLog The String to search for the runId in
+     * @return
+     */
+    private String findWorkflowId(String runLog) {
+        // This is pretty ugly, but we don't present the workflow ID in an clean format.
+        int runIdIndex = runLog.indexOf(RUN_ID_PATTERN_PREFIX);
+        int newlineFromRunId = runLog.indexOf("\n", runIdIndex);
+        return runLog.substring(runIdIndex + RUN_ID_PATTERN_PREFIX.length(), newlineFromRunId).trim();
+    }
+
+    /**
+     * Waits for the workflow corresponding to the runId parameter to reach the 'COMPLETE' state.
+     * @param runId ID of a workflow
+     * @return true if the workflow reaches the 'COMPLETE' state in time, false otherwise
+     * @throws InterruptedException
+     */
+    private boolean waitForWorkflowState(String runId, String state) throws InterruptedException {
+        for (int i = 0; i < WAIT_COUNT; i++) {
+
+            // Check on a workflow's run status
+            String[] commandStatementStatus = new String[]{ "workflow", "wes", "status",
+                "--config", TOIL_CONFIG,
+                "--id", runId
+            };
+            Client.main(commandStatementStatus);
+
+            // wait for the provided state to be printed
+            if (systemOutRule.getLog().contains(state)) {
+                return true;
+            }
+
+            Thread.sleep(WAIT_ITER_TIME_MILLI);
+        }
+
+        return false;
+    }
+
+    @Test
+    public void testBasicLaunch1() throws InterruptedException {
+        String[] commandStatementRun = new String[]{ "workflow", "wes", "launch",
+            "--config", TOIL_CONFIG,
+            "--entry", "github.com/dockstore-testing/wes-testing/single-descriptor-with-input:main",
+            "--json", ResourceHelpers.resourceFilePath("wesIt/w1_test.json"),
+            "--inline-workflow"
+        };
+        Client.main(commandStatementRun);
+        assertTrue("A helper command should be printed to stdout when a workflow is successfully started", systemOutRule.getLog().contains("dockstore workflow wes status --id"));
+
+        final String runId = findWorkflowId(systemOutRule.getLog());
+        final boolean isSuccessful = waitForWorkflowState(runId, COMPLETED_STATE);
+
+        if (!isSuccessful) {
```

Comment:
I'm sure you know this, and if you prefer this style, that's fine, but just in case, you could rewrite the block as one line:

`assertTrue("The workflow did not succeed in time.", isSuccessful);`

Reply:
Yup, thanks for pointing this out, I guess I wrote the first one weird and all subsequent tests inherited that format.

Fixed: https://github.com/dockstore/dockstore-cli/pull/121/commits/c58211042518e6b5fd9d64f15f9e636922d73d9b